src/net/http.rs

```diff
@@ -883,6 +906,13 @@ fn keep_alive_headers<W: Write>(fd: &mut W, md: &HttpResponseMetadata) -> Result
             }
         }
     }
+    match md.canonical_stacks_tip_height {
+        Some(height) => {
+            fd.write_all(format!("Canonical-Stacks-Tip-Height: {}\r\n", height).as_bytes())
```

Comment:
This isn't a standard HTTP header, so it should start with `X-`.  Also, do we need to propagate the canonical tip's index block hash as well?  Multiple Stacks blocks can have the same height.

Reply:
The health check endpoint will be checking that the node's height is in the neighborhood of its peers' heights, so I don't think that info will be necessary.